services/localzone.c

```diff
@@ -1580,8 +1627,8 @@ local_data_answer(struct local_zone* z, struct module_env* env,
 		return local_encode(qinfo, env, edns, repinfo, buf, temp, &r, 1,
 			LDNS_RCODE_NOERROR);
 	}
-	return local_encode(qinfo, env, edns, repinfo, buf, temp, lr->rrset, 1,
-		LDNS_RCODE_NOERROR);
+	return local_encode_ede(qinfo, env, edns, repinfo, buf, temp, lr->rrset, 1,
+		LDNS_RCODE_NOERROR, LDNS_EDE_FORGED_ANSWER, "");
```

Comment:
Part of the not so simple EDE cases?

```suggestion
	return local_encode(qinfo, env, edns, repinfo, buf, temp, lr->rrset, 1,
		LDNS_RCODE_NOERROR);
```